tree/ntuple/v7/test/ntuple_serialize.cxx

```diff
@@ -623,3 +623,82 @@ TEST(RNTuple, SerializeFooter)
    EXPECT_EQ(100u, pageRange.fPageInfos[0].fNElements);
    EXPECT_EQ(7000u, pageRange.fPageInfos[0].fLocator.GetPosition<std::uint64_t>());
 }
+
+TEST(RNTuple, SerializeFooterXHeader)
```

Comment:
Also here, let's test the projected field / alias column in the extension header, too.